### ISSUE TITLE
Question about model predictions

### ISSUE THREAD

**Issue by aspasia-panagiotidou** [TPM34A/Q2_2022#13]
Hi, I have a question regarding the analysis we have to make. In case the expected value is different than the actual one, why do we attribute that to a bad school performance and not to a bad model performance? I mean why do we take it for granted that the predictions of the model are correct and then based on those "correct" predictions we come to our conclusions?

Also, since we make the predictions using the test set, is it considered correct that we have already left some of the schools (the ones used in the train set) out and that for the recommendations we take into account only a subgroup of the total number of schools?

**Comment by FGarridoV** [comment]
Thank you for this interesting question. I see where your line of thought comes from. But let me reply with some provocative questions/statements for you to think about.
- What is a bad model?
- If you think one should not use models with low prediction performance, then by extension, you are saying we should stop weather forecasting (at least for more than a couple of days in advance). Do you agree?
- All models are wrong, but some are useful. Predictive ability is just one metric to judge the usefulness of a model. Which insights does your model still provide (despite its low predictive performance)?

Related to your second thought, we used all the data to train the model. Remember that an ML model is not only to estimate the parameters (from the train set); you also have to train your hyperparameters (based on the test set). We do this to avoid overfitting. If we do not do this, the model will learn the dataset and not the generalizable rules behind it.

I am curious what you mean by: *"we make the predictions using the test set"*
